autoprof/fit/iterative.py

```diff
@@ -1,11 +1,19 @@
 # Apply a different optimizer iteratively
+#According to PEP8 conventions
+from typing import Dict,Any
 import os
-import torch
+import matplotlib.pyplot as plt
+
 import numpy as np
+import torch
+
 from time import time
```

Comment:
okay I'll resolve it
there's this rule in PEP8 convention where 
if any import
lets take an example
"from .. import AP_config"
should be "import AP_config".
I have only though about shifting the AP_config file to the required file's current dir.

Reply:
Unfortunately we can't do that since AP_config is also used in other folders, but just the `fit\` one. The only way to fix this would be to move everything into a single large folder, which would be too messy. For now we'll just have to leave `from .. import AP_config` as is